spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-#require 'codeclimate-test-reporter'
-#CodeClimate::TestReporter.start
+require 'codeclimate-test-reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.